lib/challenge.js

```diff
@@ -58,9 +79,16 @@ var some_pet;
 //    the values of `name`, `species`, `breed`, and `noise`. The constructor function should also give each pet a method called `makeNoise`,
 //    which behaves as described in Q3.
 
-var Pet;
+var Pet = function Pet (name,species,breed,noise) {
+  this.name = name;
+  this.species = species;
+  this. breed = breed;
+  this.noise = noise;
+
+}
```

Comment:
What you've got here is a perfectly good constructor function. Well done! This question was looking for you to create a method on `Pet`'s prototype so that all instances of `Pet` would have the method through it. Half credit.